### ISSUE TITLE
bytearray error with featherwing_gps

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/Adafruit_CircuitPython_FeatherWing#30]
I am trying the featherwing_gps_simpletest but am getting this error.
```
Press any key to enter the REPL. Use CTRL-D to reload.
Adafruit CircuitPython 4.0.0-beta.2-97-g0dc260058 on 2019-02-19; Adafruit Grand Central M4 Express with samd51p20
>>> 
>>> 
>>> import featherwing_gps_simpletest
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "featherwing_gps_simpletest.py", line 10, in <module>
  File "adafruit_featherwing/gps_featherwing.py", line 61, in __init__
  File "adafruit_gps.py", line 122, in send_command
ValueError: string not supported; use bytes or bytearray
```

**Comment by jerryneedell** [comment]
I can fix it with this change.
```
ithub/Adafruit_CircuitPython_FeatherWing//adafruit_featherwing/
60,61c60,61
<         self._gps.send_command(bytes('PMTK314,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0','utf-8'))
<         self._gps.send_command(bytes('PMTK220,{}'.format(update_period),'utf-8'))
---
>         self._gps.send_command(b'PMTK314,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0')
>         self._gps.send_command(b'PMTK220,{}'.format(update_period))
```
but I am surprised this did not come up in testing?

**Comment by makermelissa** [comment]
I saw this in my testing and must have pushed the file without the fix. I’ll take a look as soon as I get home. I have a fix for it that I know works.

**Comment by makermelissa** [comment]
Actually, just reread (darn small phone screens). Go ahead and push your fix if it’s working.

**Comment by jerryneedell** [comment]
Ialso was going to suggest a small chage to Doc string   https://github.com/adafruit/Adafruit_CircuitPython_FeatherWing/blob/master/adafruit_featherwing/gps_featherwing.py#L42
it refers to I2C -- should be UART, correct?

**Comment by makermelissa** [comment]
Correct. Thanks for catching that.

**Comment by makermelissa** [comment]
Fixed #30 via #31.
